README.md

```diff
@@ -0,0 +1,6 @@
+This repository contains a set of easy to use scripts that helps to provision recommended infrastructure for CKEditor Collaboration Server On-Premises. You can use the script as a reference or set up a production-ready service.
+
+[Read more about Collaboration Server](https://ckeditor.com/docs/cs/latest/onpremises/cs-onpremises/overview.html)
+
```

Comment:
Could you add here some general and generic overview of the architecture. A few words that LB is needed, etc. And add link to https://ckeditor.com/docs/cs/latest/onpremises/cs-onpremises/architecture.html . 

And in AWS ECS you can use more specific definitions for AWS.

Reply:
https://www.notion.so/Common-tickets-answers-d3fc855ba4924230a2f684403f145698#900863acd4c14967abb2b6b5eb6ecb29